en/docs/deploy-operate/observe/recipe-datadog-setup.md

```
@@ -0,0 +1,231 @@
+| `ballerinax.jaeger.samplerParam` | It is a floating value. Based on the sampler type, the effect of the sampler param varies. | `1.0` | For `const`: `0` (no sampling) or `1` (sample all spans), for `probabilistic`: `0.0` to `1.0`, for `ratelimiting`: any positive integer (rate per second). |
+| `ballerinax.jaeger.reporterFlushInterval` | The Jaeger client will be sending the spans to the agent at this interval. | `2000` | Any positive integer value. |
+| `ballerinax.jaeger.reporterBufferSize` | Queue size of the Jaeger client. | `1000` | Any positive integer value. |
+
+
+## Step 5 - Send requests
+
+Run the service and send requests to `http://localhost:8090/shop/products`.
+
+Example cURL commands:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)